static_frame/core/frame.py

```diff
@@ -5968,6 +6021,55 @@ def to_latex(self,
                 )
         return repr(self.display(config))
 
+    def to_msgpack(self) -> 'bin':
+        '''
+        Return a msgpack.
+        '''
+        import msgpack
+        import msgpack_numpy
+        def encode(obj, chain=msgpack_numpy.encode):
+            clsname = obj.__class__.__name__
+            package = obj.__class__.__module__.split('.',1)[0]
+            print('package, clsname', package, clsname)
+            if package == 'static_frame':
+                if clsname in ['Index', 'IndexBase']:
```

Comment:
Rather than match to `clsname`, can you just use the `obj` reference and match to `isinstance(obj, IndexBase)`? That way you will match to all of the derived `Index` objects, such as `IndexGO` and `IndexNanosecond`.

Reply:
100% agree. I deviated from using isinstance for the moment so I could explicitly write out all of the classes as I'm learning more of the structure. Wanted to be sure I had everything covered explicitly, then go back to covering it implicitly with isinstance when I tidy up.